### ISSUE TITLE
Create Labels for Button Functions?

### ISSUE THREAD

**Issue by RevQuixo** [slinga-homebrew/Save-Game-Copier#15]
Would be nice to see an "A" button to proceed "B" to return option at the bottom.  It is fairly intuitive as-is, but it will just help for new users to navigate.

**Comment by slinga-homebrew** [comment]
I'm torn on this. Super simple to add (the Dump Memory screen has instructions) but I feel like B = back\cancel is pretty much the standard for Saturn games.

**Comment by RevQuixo** [comment]
I agree it is pretty intuitive, it is just something that was brought up a couple of times by people using the interface for the first time via MODE.  Your call ultimately.

**Comment by slinga-homebrew** [comment]
I can add something on the main page.

**Comment by slinga-homebrew** [comment]
![instructions](https://user-images.githubusercontent.com/58894244/100417724-8b25ec00-304f-11eb-948a-18c7048a727b.png)
I made a screenshot. Honestly I don't like the way it looks. Let me know what you think. 

I'm definitely leaning towards leaving instructions out.

**Comment by slinga-homebrew** [comment]
How about if I add instructions in the README.md?
